2020/09/02/rclone-upload-to-cloud-from-cli/rclone-upload-to-cloud-from-cli.html.md

```diff
@@ -0,0 +1,239 @@
+﻿---
+author: "Ardyn Elisabeth Majere"
+title: "Rclone - upload to the cloud from your command line- And much more."
+tags: cli, linux, cloud, storage,
+---
+
+### The swiss army knife of storage.
+
+Oftentimes, cloud storage providers like Google Drive are great solutions for storing files- You can upload the data and not have to worry about maintaining a separate system to host it, with all the security hassles that can bring. However- very few of the major cloud storage providers offer a command line interface, or any official way to upload without using their web interface or closed-source binary tools, if that.
```

Comment:
@endpointardyn Do you mind linking to our previous post on the subject here? https://www.endpoint.com/blog/2019/09/30/google-drive-for-vm-images

I think your comment there that the program stopped working because Google changed how they allow programmatic API access proves your point that it's kind of a pain and readers are wise to go with something like rclone that is more cross-cloud.

Reply:
Linked